mantidproject/mantid
# Rename Qt derived classes

Great care has been taken in the restructuring of the ISIS Reflectometry GUI to ensure that anything related to Qt is kept out of all classes except those that  deliberately inherit from QObject. This PR renames Qt-derived classes to start with `Q` to make it obvious which classes are QObjects.

**To test:**

- Code review and ensure unit tests pass
- Check that the ISIS Reflectometry GUI opens ok in MantidPlot and Workbench

Refs #26535 

*This does not require release notes* because **it involves file renaming only**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
